doc/newsboat.asciidoc

```diff
@@ -1306,6 +1306,32 @@ open dialogs by selecting them and pressing "Ctrl-X".
 
 include::chapter-podcasts.asciidoc[]
 
+=== Media Content
```

Comment:
The "Macro Support" chapter ends with a description of `set browser ... ; open-in-browser; set browser ...` trick, which is a perfect segue into what you describe here. Is there a rationale for making this content a separate chapter?

Reply:
Simply because the "Podcast Support" chapter (just above mine) was also a good introduction to media content. I tried to leave it open for future contributions about images and videos, and this is why I chose to create a separate chapter for this. Please note I also linked to "Macro Support" for reference.

Reply:
Ah, got it. Upon re-reading all three chapters, I tend to agree with you on this — let's keep this content as a separate chapter.

However, I still feel like the overall arrangement could be better:

1. "Macro Support", "Podcast Support" and "Media Content" are logically tied together, but there's four unrelated chapter in between "Macro Support" and the other two;
2. within the trio, it feels wrong to describe macros, then jump to Podboat, and then return back to macros.

I think this can be fixed by moving all three chapters close to each other, and reordering them:

1. Podcast Support
2. Macro Support
3. Media Content

What do you think?

Reply:
The final rundown is:

1. Macro Support
2. Open Links with External Commands
2a. Using Browser
2b. Switching Browser for Different Tasks
2c. Separating Browser from Newsboat
3. Podcast Support
3a. Managing Audio Files with Podboat
3b. Streaming Audio Content with a Media Player